googleapiclient/discovery.py

```diff
@@ -945,24 +957,26 @@ def methodNext(self, previous_request, previous_response):
     # Retrieve nextPageToken from previous_response
     # Use as pageToken in previous_request to create new request.
 
-    if 'nextPageToken' not in previous_response or not previous_response['nextPageToken']:
+    if (nextPageTokenName not in previous_response or
+        not previous_response[nextPageTokenName]):
```

Comment:
While equivalent, I think the original makes it clearer that two distinct meaningful cases are covered here.